### ISSUE TITLE
@salesforce-ux/slds/enforce-bem-class reports problem on LWC attribute

### ISSUE THREAD

**Issue by aheber** [salesforce-ux/slds-linter#40]
An HTML file that is part of an LWC can use bind syntax such as

```html
<div class={headerClass}>Text</div>
```

this says that when rendering the LWC component, the `headerClass` property should be obtained from Javascript and merged into the `class` attribute of this component.

Something like that is being flagged by the eslint BEM rule as a violation. In reality all such dynamic markers should be ignored by this rule because they are not CSS classes. Those will be assigned by Javascript and are beyond the scope of what this rule can evaluate (I think).

This is probably not the correct way to ignore this use case but something like this does work to ensure such attributes are ignored:
```javascript
       if (classAttr && classAttr.value) {
+        if (!classAttr.startWrapper || classAttr.startWrapper.value !== '"') {
+          // this isn't a quoted attribute, skip
+          return;
+        }
```
Saying that the parser much have both found a `startWrapper` node and it must be a `"` character.

**Comment by kenamverma** [comment]
Hi @aheber, thanks for reaching out. Please check the latest version ([0.1.2](https://github.com/salesforce-ux/slds-linter/releases/tag/0.1.2)) for slds-linter. This will no longer be flagged as a violation. Please test and let us know in case of any issues. Thanks.

**Comment by kenamverma** [comment]
We're closing this issue as it's been open for a while without recent activity. If you still have questions or need any help, please don’t hesitate to reach out or open a new issue. Thank you.
